### PR TITLE
Adding 1.0.0 changelog, bumping version

### PR DESCRIPTION
**Pull request by vladikoff**

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,13 @@
+v1.0.0-rc2
+  date: 2016-03-CHANGE_ME_WOLOLOLOLO
```
Comment (vladikoff) [inline CHANGELOG]: Date pending...

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
+  changes:
+    - full list of changes is on http://gruntjs.com, please also see changes from 1.0.0-rc1.
+    - if you have a Grunt plugin that includes `grunt` in the `peerDependencies`,
+      we recommend tagging with `"grunt": "">= 0.4.0"` and publishing a new version on npm.
+    - Prevent async callback from being called multiple times. Pull #1464.
+    - Update copyright to jQuery Foundation and remove redundant headers. Fixes #1478.
+    - Update glob to 7.0.x. Fixes #1467.
+    - Removing duplicate BOM strip code. Pull #1482.
```
Comment (vladikoff) [inline CHANGELOG]: both above pending PRs

File: package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "grunt",
   "description": "The JavaScript Task Runner",
-  "version": "1.0.0-rc1",
+  "version": "1.0.0-rc2",
   "author": "Grunt Development Team (http://gruntjs.com/development-team)",
   "homepage": "http://gruntjs.com/",
   "repository": "gruntjs/grunt",
```
Comment (vladikoff) [inline package.json]: We got 
```
 "grunt-legacy-log": "1.0.0-rc1",
     "grunt-legacy-util": "1.0.0-rc1",	
```

below, we can keep that until 1.0.0 or something

File: CHANGELOG [hunk 4]
```diff
@@ -1,3 +1,13 @@
+v1.0.0-rc2
+  date: 2016-03-CHANGE_ME_WOLOLOLOLO
+  changes:
+    - full list of changes is on http://gruntjs.com, please also see changes from 1.0.0-rc1.
+    - if you have a Grunt plugin that includes `grunt` in the `peerDependencies`,
+      we recommend tagging with `"grunt": "">= 0.4.0"` and publishing a new version on npm.
```
Comment (kidwm) [inline CHANGELOG]: typo: `"grunt": "">= 0.4.0"` should be `"grunt": ">= 0.4.0"`

File: package.json (hunk 3 above)
Comment (JKAussieSkater) [inline package.json]: Sorry, just browsing from a mobile device here, but does NOT updating legacy util mean [this issue](https://github.com/gruntjs/grunt/issues/1463) doesn't get resolved/tested in rc2?

**Comment by vladikoff** [comment]
@shama anything else to add here?

**Comment by shama** [comment]
:eyes: :+1:

**Comment by JKAussieSkater** [comment]
Jquery licence?

**Comment by vladikoff** [comment]
License stays the same as `"license": "MIT",`

**Comment by JKAussieSkater** [comment]
Great 😆 
Give it a pull!

**Comment by vladikoff** [comment]
aww yesss

**Comment by JKAussieSkater** [comment]
🎉🎈🍾😍

**Comment by niemyjski** [comment]
what about a github release :)

**Pull request merged**